src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Services/StarredCompletions/StarredCompletionsAssemblyHelper.cs

```diff
@@ -109,6 +124,21 @@ internal static void InitializeInstance(string? completionsAssemblyLocation, ILo
         }
     }
 
+    private static MethodInfo GetMethodInfo(Assembly assembly, string className, string methodName)
```

Comment:
No - it still isn't part of the catalog.  And mef importing this won't work anyway - we need to pass them the brokered service provider.  Likely we should make that available to extensions at some point, then this can become a normal extension.